### PR TITLE
Update what-else-do-you-need-to-know.md

### PR DESCRIPTION
**Pull request by kirillongithub**
Minor fixes

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/security/appsec/intro/what-else-do-you-need-to-know.md?ref=d7b9bedfb3d503b0b89645f7847f9bf0f74a1164

File: security/appsec/intro/what-else-do-you-need-to-know.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ How memory in a computer works, and where data is located at any given time
 ---
 ## Revision
 
-If you'd like to learn more about application security, you should make sure you have a strong grounding on the following:  
+If you'd like to learn more about application security, you should make sure you have a strong grounding in the following:
```
Comment (nem035) [inline security/appsec/intro/what-else-do-you-need-to-know.md]: should we use "foundation" instead of "grounding"? Seems to me like a more common term

Comment (nem035) [reply security/appsec/intro/what-else-do-you-need-to-know.md]:
```suggestion
If you'd like to learn more about application security, you should make sure you have a strong foundation in the following:  
```

File: security/appsec/intro/what-else-do-you-need-to-know.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ The next topics to explore, once you have those ready:
 ---
 ## Practice
 
-If you'd like to learn more about application security, you should make sure you have a strong grounding on the following:  
+If you'd like to learn more about application security, you should make sure you have a strong grounding in the following:
```
Comment (nem035) [inline security/appsec/intro/what-else-do-you-need-to-know.md]:
```suggestion
If you'd like to learn more about application security, you should make sure you have a strong foundation in the following:  
```

File: security/appsec/intro/what-else-do-you-need-to-know.md [hunk 3]
```diff
@@ -22,9 +22,9 @@ category: how to
...
+AppSec is a diverse field that means more than just keeping current on vulnerability research. It's something everyone needs to know at some level, so that when major outbreaks occur they can take steps to protect themselves and the applications they develop. At more advanced levels, it means strictly following regulatory guidelines to ensure certain kinds of data are very well protected. At very advanced levels, it can mean research, penetration testing, and more.
 
-If you'd like to learn more about application security, you should make sure you have a strong grounding on the following:
+If you'd like to learn more about application security, you should make sure you have a strong grounding in the following:
```
Comment (nem035) [inline security/appsec/intro/what-else-do-you-need-to-know.md]:
```suggestion
If you'd like to learn more about application security, you should make sure you have a strong foundation in the following:
```

**Pull request merged**